extra_docs/sensitivities/covariance_sensitivity.tex

```diff
@@ -0,0 +1,255 @@
+\documentclass[11pt, oneside]{article}   	% use "amsart" instead of "article" for AMSLaTeX format
+\usepackage{geometry}                		% See geometry.pdf to learn the layout options. There are lots.
+\geometry{letterpaper}                   		% ... or a4paper or a5paper or ... 
+%\geometry{landscape}                		% Activate for rotated page geometry
+%\usepackage[parfill]{parskip}    		% Activate to begin paragraphs with an empty line rather than an indent
+\usepackage{graphicx}				% Use pdf, png, jpg, or eps§ with pdflatex; use eps in DVI mode
+								% TeX will automatically convert eps --> pdf in pdflatex		
+\usepackage{amssymb, amsmath, amsthm, bbm}
+						% Activate to display a given date or no date
+
+\usepackage{macros}
+
+\begin{document}
+
+\begin{definition}
+Let $X$ and $Y$ be two column vectors of length n. Then the sample covariance between $X$ and $Y$, $\cov[X,Y]$, is defined to be 
+$$ \cov[X,Y] = \frac{1}{n-1} \sum_{i=1}^n (x_i - \mu_X)(y_i - \mu_Y),$$
+where $\mu_X$ is the mean of $X$ and $\mu_Y$ is the mean of $Y$.
+\end{definition}
+
+\begin{definition}
+Let 
+$$
+X = 
+\begin{bmatrix}
+X_1 & X_2 & \cdots & X_m
+\end{bmatrix}^T,
```

Comment:
I think transposing here is consistent with the definition of the covariance matrix below, I'm just confused as to why we are transposing in the first place if we are suppose to view X_i as column vectors.

Reply:
I defined them as column vectors since that's how they are implemented in the library.